src/quo2/components/tags/number_tag/view.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.tags.number-tag.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [quo2.components.tags.number-tag.style :as style]
+            [quo2.theme :as quo.theme]))
+
+(defn number-tag-internal
+  [{:keys [number size-key blur? theme] :as props}]
```

Comment:
imo it's better the api matches figma and is `size` externally - it's also like this for all other comoponents 👍

Reply:
done